controllers/controllers.js

```diff
@@ -0,0 +1,9 @@
+const { response } = require("../app");
+const { selectTopics } = require("../models/models");
+
+exports.getTopics = (request, response) => {
+    selectTopics().then((topic) => {
+        // console.log(topics)
```

Comment:
Make sure you remove console.logs once you're happy the info is coming back as you expect it to.